mappings/net/minecraft/block/CandleBlock.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/class_5544 net/minecraft/block/CandleBlock
+	FIELD field_27174 CANDLES Lnet/minecraft/class_2758;
+	FIELD field_27175 LIT_2 Lnet/minecraft/class_2746;
```

Comment:
```suggestion
	FIELD field_27175 LIT Lnet/minecraft/class_2746;
```

Reply:
Fixed in 48c14bf52cc9898db151ecdd2b02c0851839a51a